src/pages/Home/components/MovieAddButton/MovieAddButton.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Button from "@material-ui/core/Button";
+
+const movieAddButtonStyle = {
+    backgroundColor: 'white',
+    color: '#f65261',
+    width: '120px',
+    fontSize: '1.2rem',
+}
+
```

Comment:
Could you please move the style to MovieAddButton.scss  class?

Reply:
Thanks, the style was moved to MovieAddButton.scss class.